app/src/main/java/org/schabi/newpipe/info_list/holder/CommentsMiniInfoItemHolder.java

```diff
@@ -53,20 +56,25 @@
     private final Linkify.TransformFilter timestampLink = new Linkify.TransformFilter() {
         @Override
         public String transformUrl(final Matcher match, final String url) {
-            int timestamp = 0;
-            final String hours = match.group(1);
-            final String minutes = match.group(2);
-            final String seconds = match.group(3);
-            if (hours != null) {
-                timestamp += (Integer.parseInt(hours.replace(":", "")) * 3600);
-            }
-            if (minutes != null) {
-                timestamp += (Integer.parseInt(minutes.replace(":", "")) * 60);
-            }
-            if (seconds != null) {
-                timestamp += (Integer.parseInt(seconds));
+            try {
+                int timestamp = 0;
+                final String hours = match.group(1);
+                final String minutes = match.group(2);
+                final String seconds = match.group(3);
+                if (hours != null) {
+                    timestamp += (Integer.parseInt(hours.replace(":", "")) * 3600);
+                }
+                if (minutes != null) {
+                    timestamp += (Integer.parseInt(minutes.replace(":", "")) * 60);
+                }
+                if (seconds != null) {
+                    timestamp += (Integer.parseInt(seconds));
+                }
+                return streamUrl + url.replace(match.group(0), "#timestamp=" + timestamp);
+            } catch (final Exception ex) {
+                Log.d(TAG, "Unable to process url='" + url + "' as timestampLink", ex);
```

Comment:
I think this would also be good as an error
```suggestion
                Log.e(TAG, "Unable to process url='" + url + "' as timestampLink", ex);
```